runtime/onert/core/src/compiler/train/TrainingCompiler.cc

```diff
@@ -194,8 +208,8 @@ std::shared_ptr<CompilerArtifact> TrainingCompiler::compile(void)
   for (const auto &pair : lowered_subgs)
   {
     const auto &subg_index = pair.first;
-    const auto &lowered_subg = pair.second;
-    dot_dumper.dump(*lowered_subg, nnfw::misc::str("after_lower_subg-", subg_index.value()));
+    const auto &subg = pair.second;
+    dot_dumper.dump(*subg, nnfw::misc::str("after_lower_subg-", subg_index.value()));
```

Comment:
It would be good to factor out this repeated code as a function. `TrainingCompiler.cc` is not the only place, same pattern are found in `Compiler.cc`. It is out of the scope of this PR.